src/node/interfaces.cpp

```diff
@@ -169,6 +170,27 @@ class NodeImpl : public Node
     }
     void mapPort(bool use_upnp, bool use_natpmp) override { StartMapPort(use_upnp, use_natpmp); }
     bool getProxy(Network net, Proxy& proxy_info) override { return GetProxy(net, proxy_info); }
+    std::string defaultProxyAddress() override
+    {
+        return std::string(DEFAULT_PROXY_HOST) + ":" + ToString(DEFAULT_PROXY_PORT);
+    }
+    bool validateProxyAddress(const std::string& addr_port) override
```

Comment:
validateProxyAddress shouldn't be in this module. It should exist in the nodemodel or possibly in netbase.

Reply:
I agree, maybe it could be moved deeper into de netbase but I didn't what to touch that at this stage, it can be done later once we update the interfaces upstream.